airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -2000,3 +2063,87 @@ schemas:
           - "null"
           - string
         format: date-time
+
+  contact_lists:
+    $schema: http://json-schema.org/draft-07/schema#
+    type:
+      - "null"
+      - object
+    additionalProperties: true
+    properties:
+      processingType:
+        type:
+          - "null"
+          - string
+      objectTypeId:
+        type:
+          - "null"
+          - string
+      updatedById:
+        type:
+          - "null"
+          - string
+      filtersUpdatedAt:
+        type:
+          - "null"
+          - string
+        format: date-time
+      listId:
+        type:
+          - "null"
+          - string
+      createdAt:
+        type:
+          - "null"
+          - integer
```

Comment:
as mentioned above, we should make this and `updatedAt` strings since we'll no longer be doing the transformation